ministryofjustice/correspondence_tool_staff
# CT-2580 Add cases warehouse job to all environments

## Description
Executes `./cts cases warehouse` every 30 minutes to ensure `warehouse_case_reports` is consistent.
 
## Self-review checklist
 
### Screenshots
N/A
 
### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2580
 
### Deployment
Deploy as per current process to all environments
 
### Manual testing instructions
1. In non-production environment `truncate table warehouse_case_reports;`
2. Download Case Closed report - should be empty.
3. Wait 30 minutes.
4. Download Case Closed report - should not be empty.
